src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+import { getSortedPeople } from '../../utils/getSortedPeople';
+import { SortLink } from '../SortLink';
+
+type Props = {
+  people: Person[],
+  personSlug: string,
+};
+
+export const PeopleTable: React.FC<Props> = React.memo(({
+  people,
+  personSlug,
+}) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') as keyof Person;
+  const isReversed = searchParams.get('order') === 'desc';
+  const query = searchParams.get('query');
+  const sexFilter = searchParams.get('sex');
```

Comment:
It's more likely to name this way 
```suggestion
  const sex = searchParams.get('sex') || '';
```